static/js/admin.js

```diff
@@ -516,6 +518,9 @@ function _setup_page() {
         if (e.which === 13) {
             e.preventDefault();
             e.stopPropagation();
+            var stream_name = $(".create_default_stream").val();
+            make_stream_default(stream_name);
+            $(".create_default_stream")[0].value = "";
```

Comment:
Can't you just set a variable to `$(".create_default_stream")` and then use it to reduce DOM access?